spec/temporal.html

```diff
@@ -1,5 +1,5 @@
-<!doctype html>
-<meta charset="utf8">
+<!DOCTYPE html>
+<meta charset="utf8" />
```

Comment:
```suggestion
<meta charset="utf8">
```

Reply:
Suggested revert is fine with me because the missing slash didn't break anything before, but this change and the line above it was automatically made by prettier. Is prettier wrong here?

Reply:
the trailing slash isn't required in html5, but personally i find it a much better style to always include it on a self-closing element.

Reply:
It's meaningless cruft, and inconsistent with the other files.